pkg/querier/blocks_store_queryable.go

```diff
@@ -626,14 +627,18 @@ func filterBlocksByShard(blocks bucketindex.Blocks, queryShardIndex, queryShardC
 	return blocks, incompatibleBlocks
 }
 
-// canBlockWithCompactorShardIndexContainQueryShard returns false if block with given compactor shard ID can *definitely NOT*
-// contain series for given query shard. Returns true otherwise (we don't know if block *does* contain such series,
+// canBlockWithCompactorShardIndexContainQueryShard returns false if block with
+// given compactor shard ID can *definitely NOT* contain series for given query shard.
+// Returns true otherwise (we don't know if block *does* contain such series,
 // but we cannot rule it out).
 //
-// In other words, if this function returns false, block with given compactorShardID doesn't need to be searched for series from given query shard.
+// In other words, if this function returns false, block with given compactorShardID
+// doesn't need to be searched for series from given query shard.
 //
-// In addition this function also returns whether query and compactor shard counts were divisible by each other (one way or the other).
-func canBlockWithCompactorShardIndexContainQueryShard(queryShardIndex, queryShardCount, compactorShardIndex, compactorShardCount uint64) (result bool, divisibleShardCounts bool) {
+// In addition this function also returns whether query and compactor shard counts were
+// divisible by each other (one way or the other).
+func canBlockWithCompactorShardIndexContainQueryShard(queryShardIndex, queryShardCount, compactorShardIndex,
+	compactorShardCount uint64) (result bool, divisibleShardCounts bool) {
```

Comment:
I don't find this being the prettiest formatting, but we don't have a linter for it so... :shrug:

Reply:
We experimented with having a lint for this in enterprise metrics and it was:

* Very destructive in terms of `git blame`
* Annoying